src/examples/HelloSPTP.cpp

```diff
@@ -38,9 +42,17 @@ namespace examples {
 using namespace std;
 using namespace nupic;
 using namespace nupic::utils;
+
 using nupic::ScalarEncoder;
+
 using nupic::algorithms::spatial_pooler::SpatialPooler;
-using nupic::algorithms::Cells4::Cells4;
+
+using TP =     nupic::algorithms::Cells4::Cells4;
+using BackTM = nupic::algorithms::backtracking_tm::BacktrackingTMCpp;
+using namespace nupic::algorithms::temporal_memory;
```

Comment:
for some reason `nupic::algorithms::temporal_memory` is not a name-space, when it clearly is?!?

```
[ 59%] Building CXX object src/test/CMakeFiles/unit_tests.dir/unit/types/SdrTest.cpp.o
In file included from /home/mmm/devel/HTM/htm-community/nupic.cpp/src/examples/Hotgym.cpp:1:
/home/mmm/devel/HTM/htm-community/nupic.cpp/src/examples/HelloSPTP.cpp:45:36: error: ‘temporal_memory’ is not a namespace-name
using namespace nupic::algorithms::temporal_memory;
^~~~~~~~~~~~~~~
/home/mmm/devel/HTM/htm-community/nupic.cpp/src/examples/HelloSPTP.cpp:45:51: error: expected namespace-name before ‘;’ token
using namespace nupic::algorithms::temporal_memory;
^
/home/mmm/devel/HTM/htm-community/nupic.cpp/src/examples/HelloSPTP.cpp:53:36: error: ‘temporal_memory’ is not a namespace-name
using namespace nupic::algorithms::temporal_memory;
^~~~~~~~~~~~~~~
/home/mmm/devel/HTM/htm-community/nupic.cpp/src/examples/HelloSPTP.cpp:53:51: error: expected namespace-name before ‘;’ token
using namespace nupic::algorithms::temporal_memory;
^```

Reply:
missing include, I felt really stupid!